project/MiMa.scala

```diff
@@ -43,39 +45,18 @@ object MiMa extends AutoPlugin {
       Set.empty
     } else {
       val versions: Seq[String] = {
-        val firstPatchOf25 =
-          if (scalaBinaryVersion.startsWith("2.13")) 25
-          else if (projectName.contains("discovery")) 19
-          else if (projectName.contains("coordination")) 22
-          else 0
+        val akka26Previous = expandVersions(2, 6, firstPatchOf26 to latestPatchOf26)
+        // FIXME during milestone phase we check latest milestone but when final it should be replaced with
+        // val akka27Previous = expandVersions(2, 7, 0 to latestPatchOf27)
+        // FIXME why can't the snapshot be used here? List("2.7.0-M1")
+        val akka27Previous = Nil
```

Comment:
I don't know why it fails when including the M1. It's then missing jdk9+ classes. I have check that the class files exists in the published jars.
```
[error]  * class akka.stream.javadsl.JavaFlowSupport#Source does not have a correspondent in current version
[error]    filter with: ProblemFilters.exclude[MissingClassProblem]("akka.stream.javadsl.JavaFlowSupport$Source")
```

```
[error]  * class akka.actor.typed.internal.jfr.DeliveryConsumerStarted does not have a correspondent in current version
[error]    filter with: ProblemFilters.exclude[MissingClassProblem]("akka.actor.typed.internal.jfr.DeliveryConsumerStarted")
```